services/shhext/chat/crypto/crypto.go

```diff
@@ -0,0 +1,91 @@
+package crypto
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/rand"
+	"errors"
+)
+
+const (
+	aesNonceLength = 12
+)
+
+func EncryptSymmetric(key []byte, plaintext []byte) ([]byte, error) {
```

Comment:
Tip: if the arguments have the same type, you can group them like this `key, plaintext []byte`.